### DISCUSSION TITLE
How change type of vertex in android version?

### DISCUSSION THREAD

**Discussion by Makar0ff** [OpenOrienteering/mapper#1746] [Questions & Answers] [answered]
### Steps to reproduce
1.How I can change type of vertex from broken line to spline? In android version?
2.
3.

### Actual behaviour
Tell us what happens

### Expected behaviour
Tell us what should happen instead

### Configuration
Mapper Version:
Operating System:

**Comment by mlerjen** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/15611653/95326407-d2d79500-08a2-11eb-8d6c-a312803ddde9.png)
Think of changing a straight line into a curve instead. Work therefore with the edit line tool. Crtl& click on the line will switch the segment between straight line and curve and vice versa. The proceeding is similar for the Android version.

**Comment by dg0yt** [comment]
On Android, it is nearly the same, however, you will need to activate "Toggle curve" instead of using the Ctrl key.

![toggle-curve](https://user-images.githubusercontent.com/13567791/95492929-0d712880-099c-11eb-851f-1631aeaf6e16.png)

**Comment by Makar0ff** [comment]
Thank you!
